tests/unit/dataactvalidator/test_c19_award_financial.py

```diff
@@ -0,0 +1,105 @@
+                         main_account_code='1', sub_account_code='1', object_class='2', program_activity_code='1',
+                         parent_award_id='1', piid='1', uri='1', fain='1', transaction_obligated_amou='1')
+
+    af10 = AwardFinancial(job_id=1, row_number=1, beginning_period_of_availa='1', ending_period_of_availabil='1',
+                         agency_identifier='1', allocation_transfer_agency='1', availability_type_code='1',
+                         main_account_code='1', sub_account_code='1', object_class='1', program_activity_code='2',
+                         parent_award_id='1', piid='1', uri='1', fain='1', transaction_obligated_amou='1')
+
+    af11 = AwardFinancial(job_id=1, row_number=1, beginning_period_of_availa='1', ending_period_of_availabil='1',
+                         agency_identifier='1', allocation_transfer_agency='1', availability_type_code='1',
+                         main_account_code='1', sub_account_code='1', object_class='1', program_activity_code='1',
+                         parent_award_id='2', piid='1', uri='1', fain='1', transaction_obligated_amou='1')
+
+    af12 = AwardFinancial(job_id=1, row_number=1, beginning_period_of_availa='1', ending_period_of_availabil='1',
+                         agency_identifier='1', allocation_transfer_agency='1', availability_type_code='1',
+                         main_account_code='1', sub_account_code='1', object_class='1', program_activity_code='1',
+                         parent_award_id='1', piid='2', uri='1', fain='1', transaction_obligated_amou='1')
+
+    af13 = AwardFinancial(job_id=1, row_number=1, beginning_period_of_availa='1', ending_period_of_availabil='1',
+                         agency_identifier='1', allocation_transfer_agency='1', availability_type_code='1',
+                         main_account_code='1', sub_account_code='1', object_class='1', program_activity_code='1',
+                         parent_award_id='1', piid='1', uri='2', fain='1', transaction_obligated_amou='1')
+
+    af14 = AwardFinancial(job_id=1, row_number=1, beginning_period_of_availa='1', ending_period_of_availabil='1',
+                         agency_identifier='1', allocation_transfer_agency='1', availability_type_code='1',
+                         main_account_code='1', sub_account_code='1', object_class='1', program_activity_code='1',
+                         parent_award_id='1', piid='1', uri='1', fain='2', transaction_obligated_amou='1')
+
+    af15 = AwardFinancial(job_id=1, row_number=1, beginning_period_of_availa='1', ending_period_of_availabil='1',
+                          agency_identifier='1', allocation_transfer_agency='1', availability_type_code='1',
+                          main_account_code='1', sub_account_code='1', object_class='1', program_activity_code='1',
+                          parent_award_id='1', piid='1', uri='1', fain='1', transaction_obligated_amou='2')
+
```

Comment:
Similar to above, in this case the typical usage is that only one of (piid, fain, uri) will be populated, so it would be good to have a test that covered that case.